dnpchallenge/docker-compose.yml

```diff
@@ -1,26 +1,17 @@
 services:
   rtu:
-    container_name: otsim-rtu
-    image: ghcr.io.patsec/ot-sim/ot-sim:main
-    # image: ot-sim:test
+    image: ghcr.io/patsec/ot-sim/ot-sim:main
+    privileged: true
     volumes:
-      - ./rtuconfig/xml:/etc/otsim/config.xml
+      - ./rtuconfig.xml:/etc/ot-sim/config.xml
```

Comment:
The ot-sim Docker image automatically runs the `ot-sim-cpu-module` at startup and looks for a default config at `/etc/ot-sim/config.xml`.